diff -uNr mercurial-2.0.1/mercurial/demandimport.py mercurial-2.0.1.new/mercurial/demandimport.py
--- mercurial-2.0.1/mercurial/demandimport.py	2011-12-01 22:55:32.000000000 +0100
+++ mercurial-2.0.1.new/mercurial/demandimport.py	2011-12-04 12:34:49.562109291 +0100
@@ -144,6 +144,7 @@
     '_ssl', # conditional imports in the stdlib, issue1964
     'rfc822',
     'mimetools',
+    'gobject._gobject',
     ]
 
 def enable():
