--- mercurial-3.5.2/mercurial/demandimport.py~	2015-10-01 19:06:55.000000000 +0200
+++ mercurial-3.5.2/mercurial/demandimport.py	2015-10-27 10:27:30.377611999 +0100
@@ -173,7 +173,8 @@ ignore = [
     'rfc822',
     'mimetools',
     # setuptools 8 expects this module to explode early when not on windows
-    'distutils.msvc9compiler'
+    'distutils.msvc9compiler',
+    'gobject._gobject'
     ]
 
 def isenabled():
